site-docker.yml.sample

```diff
@@ -70,6 +70,27 @@
         - (ceph_docker_dev_image is undefined or not ceph_docker_dev_image)
         - (not (inventory_hostname in groups.get('clients', [])) or (inventory_hostname == groups.get('clients', [''])|first))
 
+- hosts:
+  - mons
+  - agents
+  - osds
+  - mdss
+  - rgws
+  - nfss
+  - restapis
+  - rbdmirrors
+  - clients
+  - mgrs
+  - iscsi-gws
+
+  gather_facts: false
+  any_errors_fatal: true
+  become: true
+
+  roles:
+    - ceph-defaults
+    - ceph-validate
```

Comment:
The validation should be the first thing we do, here we potentially try to fetch container images first and then run the validation.

Reply:
That container fetching happens in ``ceph-defaults``? I do need the ``ceph-defaults`` role before ``ceph-validate`` of the action plugin doesn't have access to the variables I'm trying to validate. I ran into issues when I removed ``ceph-defaults``.

Reply:
@leseb it's sort of what we have been already facing before coming up with this validate module.